src/renderer/components/header/netstatus/HeaderNetStatus.stories.tsx

```diff
@@ -7,17 +7,51 @@ import { HeaderNetStatus } from './HeaderNetStatus'
 
 const binanceUrl = O.some('https://testnet-dex.binance.org/api/v1')
 const midgardUrl = O.some('https://54.0.0.27')
+const bitcoinUrl = O.some('https://blockstream.info')
+const thorchainUrl = O.some('https://thorchain.net')
 
 storiesOf('Components/HeaderNetStatus', module)
   .add('default', () => {
-    return <HeaderNetStatus isDesktopView={true} binanceUrl={binanceUrl} midgardUrl={midgardUrl} />
+    return (
+      <HeaderNetStatus
+        isDesktopView={true}
+        binanceUrl={binanceUrl}
+        midgardUrl={midgardUrl}
+        bitcoinUrl={bitcoinUrl}
+        thorchainUrl={thorchainUrl}
+      />
+    )
```

Comment:
[hint] to avoid duplications, create `defaultProps` and use/change it for all stories (check other stories, eg. [SymDeposit.stories](https://github.com/thorchain/asgardex-electron/blob/e9f5e8b000f6d5bf2f1f97ec5a1710ca9e33d135/src/renderer/components/deposit/add/SymDeposit.stories.tsx))